diff --git a/yandextank/core/config/00-base.ini b/yandextank/core/config/00-base.ini
index b56fba1..5734123 100644
--- a/yandextank/core/config/00-base.ini
+++ b/yandextank/core/config/00-base.ini
@@ -5,7 +5,7 @@ plugin_ShellExec=yandextank.plugins.ShellExec
 plugin_phantom=yandextank.plugins.Phantom
 plugin_aggreg=yandextank.plugins.Aggregator
 plugin_autostop=yandextank.plugins.Autostop
-plugin_monitoring=yandextank.plugins.Monitoring
+#plugin_monitoring=yandextank.plugins.Monitoring
 plugin_console=yandextank.plugins.ConsoleOnline
 plugin_tips=yandextank.plugins.TipsAndTricks
 plugin_totalautostop=yandextank.plugins.TotalAutostop
